commands/build.go

```diff
@@ -521,3 +549,49 @@ func (s *shmSize) Set(v string) error {
 func (s *shmSize) Type() string {
 	return s.org.Type()
 }
+
+func addDebugShellCommand(cmd *cobra.Command, dockerCli command.Cli) {
+	cmd.AddCommand(
+		debugShellCmd(dockerCli),
+	)
+}
```

Comment:
I think the issue with `docs-upstream` workflow is because command is not added to the root one so it would look for `docker buildx build debug-shell`. I will take a closer look.

Reply:
Is `debug-shell` supposed to be a sub command of `build`?

Reply:
@crazy-max Thank you for looking at this.

> Is debug-shell supposed to be a sub command of build?

No, it' supposed to be a root command (`docker buildx debug-shell`) and it's actually implemented as a root command as described in the above PR description.